src/qt/guiutil.cpp

```diff
@@ -749,6 +749,19 @@ QString boostPathToQString(const fs::path &path)
     return QString::fromStdString(path.string());
 }
 
+QString ConnectionTypeToQString(ConnectionType conn_type)
+{
+    switch (conn_type) {
+    case ConnectionType::INBOUND: return "Inbound full relay";
+    case ConnectionType::OUTBOUND_FULL_RELAY: return "Outbound full relay";
+    case ConnectionType::BLOCK_RELAY: return "Outbound block relay";
+    case ConnectionType::MANUAL: return "Outbound manual";
+    case ConnectionType::FEELER: return "Outbound feeler";
+    case ConnectionType::ADDR_FETCH: return "Outbound address fetch";
+    } // no default case, so the compiler can warn about missing cases
+    assert(false);
+}
+
```

Comment:
571b0469fa952be98874d1afd772a7fa708dd83f

The returned values should be translated.

UX note: I have a personal preference do not capitalize the first letter of returned values. Feel free to ignore it.

Reply:
I also prefer all lowercase (e.g. rpc/cli) but the GUI UI uses capitalization (I hesitated to use titlecase here).

Reply:
I perceive it like "Connection type: Outbound full relay". That's why I prefer the lowercased variant.

If the capitalized variant will be required in some other places in the GUI, it is easy to provide it with a generic function.

I'm not a design expert, then I'm fine with any variant.

Reply:
Thanks--translated and changed to titlecase for coherence with the general GUI UI.